### PR TITLE
HMS-7405 Add reference for active rooms get peers API

### PR DESCRIPTION
**Pull request by pratishbodhale**

### CODE REVIEW

File: docs/server-side/v2/active-rooms/get-peers.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
+nav: 3.9
+---
+
+You can use this API for get the active peers in the room. Peers can be filtered by specifying user_id or role in the query params
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/get-peers.mdx:6]: please update user_id and role with backticks as a blockquote.

File: docs/server-side/v2/active-rooms/get-peers.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+```
+
+</ResponseBox>
+
+## Allowed filters
+
+| Name          | Type      | Description                                                       | Required |
+| :------------ | :-------- | :-----------------------------------------------------------------| :------- |
+| user_id <br/> | `string`  | Determines the peers with specified user_id                       | No       |
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/get-peers.mdx:48]: shall we add some info about how the user_id is generated. 
refer to [this](https://www.100ms.live/docs/server-side/v2/Sessions/example-build-attendance#calculate-the-individual-participants-duration).

File: docs/server-side/v2/active-rooms/get-peers.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/get-peers.mdx:2]: shall we update this to List peers to maintain consistency with other list APIs.

File: docs/server-side/v2/active-rooms/get-peers.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+nav: 3.9
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/get-peers.mdx:3]: Also, move this after `Retrieve a specific peer` in the sidebar. You can update this to 3.5 and bump the decimal points for Update a peer, etc.

File: docs/server-side/v2/active-rooms/get-peers.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+</Request>
+<ResponseBox id="res-0" status="200 OK">
+
+```json
+{
+    "peers": {
+        "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20": {
+            "id": "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20",
+            "name": "John",
+            "user_id": "da3136fd-622e-468f-96ca-09c8c9f50072",
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/get-peers.mdx:25]: shall we use some intuitive values for user_id field here, maybe something like student1 & student2?

**Review by Nixon-Samuel — CHANGES_REQUESTED** [review]

File: docs/server-side/v2/active-rooms/get-peers.mdx (hunk 5 above)
Comment (pratishbodhale) [reply docs/server-side/v2/active-rooms/get-peers.mdx:25]: sure, I'll put something like `student-637`, `student-7810`

File: docs/server-side/v2/active-rooms/get-peers.mdx (hunk 2 above)
Comment (pratishbodhale) [reply docs/server-side/v2/active-rooms/get-peers.mdx:48]: I can add this (your internal user identifier used while creating the [auth token](https://www.100ms.live/docs/server-side/v2/introduction/authentication-and-tokens#app-token) for peers).

File: docs/server-side/v2/active-rooms/list-peers.mdx [hunk 6]
```diff
@@ -22,31 +22,30 @@ curl --location --request GET 'https://api.100ms.live/v2/active-rooms/<room_id>/
...
 
 ## Allowed filters
 
-| Name          | Type      | Description                                                       | Required |
-| :------------ | :-------- | :-----------------------------------------------------------------| :------- |
-| user_id <br/> | `string`  | Determines the peers with specified user_id                       | No       |
-| role <br/>    | `string`  | Determines the peers with specified role                          | No       |
+| Name          | Type      | Description                                                                                                               | Required |
+| :------------ | :-------- | :-------------------------------------------------------------------------------------------------------------------------| :------- |
+| user_id <br/> | `string`  | Determines the peers with specified user_id ((your internal user identifier used while creating the auth token for peers) | No       |
```
Comment (Nixon-Samuel) [inline docs/server-side/v2/active-rooms/list-peers.mdx:47]: (( - typo

**Review by Nixon-Samuel — APPROVED** [review]

**Review by sagargaonkar — APPROVED** [review]

**Pull request merged**